assets/admin_themes/Sea_Green/layout/body.scss

```
@@ -5,4 +5,34 @@ body {
+a:focus,
+button:focus,
+input:focus,
+select:focus,
+textarea:focus,
+summary:focus,
+details:focus,
+[tabindex]:not([tabindex="-1"]):focus,
+[contenteditable]:not([contenteditable="false"]):focus,
+iframe:focus,
+area:focus {
+    outline: 2px solid currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)